src/log-analytics/azext_loganalytics/_help.py

```diff
@@ -7,22 +7,29 @@
 
 # pylint: disable=line-too-long
 
+helps['monitor log-analytics'] = """
+    type: group
+    short-summary: Commands for querying data in Log Analytics workspaces.
+"""
+
 helps['monitor log-analytics query'] = """
     type: command
     short-summary: Query a Log Analytics workspace.
     parameters:
-        - workspace: --workspace -w
-          type: string
-          short-summary: GUID of the Log Analytics workspace.
-        - analytics-query: --analytics-query
-          type: string
-          short-summary: Query to execute over the Log Analytics data.
-        - timespan: --timespan -t
-          type: string
-          short-summary: Timespan over which to query data. Defaults to all data.
-        - workspaces: --workspaces
-          type: array
-          short-summary: Additional workspaces to union data for querying. Specify additional workspace IDs separated by commas.
+      - name: --workspace
```

Comment:
No need to add help for the parameters here as they will be gotten from [_params.py](https://github.com/Azure/azure-cli-extensions/blob/90a71cac2d7b6915fa5d73d8f2d7e11699e6c9d7/src/log-analytics/azext_loganalytics/_params.py). It can be beneficial to add help for parameters here if you need to add more content like [long summaries or populator-commands](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_help.md).  

You don't need to remove the help you added here, just let them be consistent with the help in your _params.py file.

Reply:
This did not work 100% for me. --analytics-query did not load any text from the parameter python file (or elsewhere). Did work for other parameters, though.